flows/aws/interactive-workflow/README.md

```diff
@@ -20,6 +20,7 @@ Interactive Workflow Demo is designed to guide you through the process of settin
 Before starting, ensure you have the following installed:
 - Python 3.6+
 - Prefect
+- Marvin 
 - Any other dependencies listed in `requirements.txt`.
 
 ### Installation
```

Comment:
"Installation" and "Running the Demo" could be consolidated into a single "Setup" section.